testing/integration-tests/src/events/mod.rs

```diff
@@ -4,47 +4,49 @@
 
 use crate::{
     node_runtime::{
+        self,
         balances,
         system,
     },
     pair_signer,
     test_context,
+    utils::wait_for_blocks,
 };
 use futures::StreamExt;
 use sp_keyring::AccountKeyring;
 
 // Check that we can subscribe to non-finalized block events.
 #[tokio::test]
-async fn non_finalized_block_subscription() -> Result<(), subxt::BasicError> {
-    tracing_subscriber::fmt::try_init().ok();
```

Comment:
Yeah; I got rid of them because the test output became unnecessarily verbose (but left the crate so that we could add it back on a case-by-case basis for debugging tests)